app/templating/summary/question.py

```diff
@@ -0,0 +1,76 @@
+from app.questionnaire.navigator import evaluate_rule
+from app.templating.summary.answer import Answer
+
+
+class Question:
+
+    def __init__(self, block_id, question_schema, answers):
+        self.id = question_schema['id']
+        self.type = question_schema['type']
+        self.skip_condition = question_schema.get('skip_condition')
+        answer_schema = question_schema['answers']
+        self.title = question_schema['title'] or answer_schema[0]['label']
+        self.answers = Question._build_answers(block_id, question_schema, answer_schema, answers)
+
+    def is_skipped(self, answers):
```

Comment:
The question only holds reference to its own answers, we could have reference to all answers but I think we'd be holding state for the sake of the function instead of passing it